docs/components/camera/transform.md

```diff
@@ -96,6 +96,32 @@ Classifications overlay text from the `GetClassifications` method of the [Vision
 
 {{% /tab %}}
 
+{{% tab name="Crop" %}}
+
+The Crop transform crops takes an image and crops it to a rectangular area specified by two points: `(x_min, y_min)` and `(x_max, y_max)`.
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "type": "crop",
+    "attributes": {
+        "x_min_px": int,
+        "y_min_px": int,
+        "x_max_px": int,
+        "y_max_px": int
+    }
+}
+
+```
+
+**Attributes:**
+
+- `x_min_px`: The x coordinate of the first point that specifies the rectangular area to crop the image to.
+- `y_min_px`: The y coordinate of the first point that specifies the rectangular area to crop the image to.
+- `x_max_px`: The x coordinate of the second point that specifies the rectangular area to crop the image to.
+- `y_max_px`: The y coordinate of the second point that specifies the rectangular area to crop the image to.
```

Comment:
[optional] If it's clearer, the first point is the "top left" point of the cropping rectangle, and the second point would be the "bottom right" point of the cropping rectangle.

Reply:
Thank you - I was gonna ask that. I tried to test it and then realized it's not merged yet - so I'll chagne that and then just let me know when it's good to go :)